app/views/dashboards/index.html.erb

```diff
@@ -2,8 +2,8 @@
   <div class="col-md-12">
     <%= render partial: 'dashboards/overview' %>
     <%= render partial: 'dashboards/search_form' %>
-    <%= render partial: 'dashboards/table_content', locals: { title: 'Your call list', table_type: 'call_list', pregnancies: current_user.pregnancies } %>
-    <%= render partial: 'dashboards/table_content', locals: { title: 'Your completed calls', table_type: 'completed_calls', pregnancies: current_user.pregnancies } %>
+    <%= render partial: 'dashboards/table_content', locals: { title: 'Your call list', table_type: 'call_list', pregnancies: call_list_pregnancies } %>
```

Comment:
If we change those helper methods to model methods, it'll necessitate changing this around -- just a flag, just in case.